cmd/gcp-routes-controller/README.md

```diff
@@ -0,0 +1,23 @@
+# gcp-routes-controller
+
+## Background
+
+Google cloud load balancer is a L3LB that is Special. It doesn't do DNAT; instead, it
```

Comment:
nit: special

Reply:
hah, needless capitalization is classic Nerd English but I'll change it :p